packages/create-app/src/commands/init.ts

```diff
@@ -0,0 +1,28 @@
+import {execSync} from 'child_process';
+
+import {Command, Flags} from '@oclif/core';
+import path from 'pathe';
+
+export default class InitCommand extends Command {
+  static description = 'Create new GestaltJS app';
+  static flags = {
+    name: Flags.string({
+      char: 'n',
+      description: 'project name',
+    }),
+    path: Flags.string({
+      char: 'p',
+      description: 'project path',
+    }),
+  };
+
+  async run(): Promise<void> {
+    const {args, flags} = await this.parse(InitCommand);
+    const name = flags.name ? flags.name : 'my-gestalt-project';
+    const projectPath = flags.path ? flags.path : this.config.home;
```

Comment:
When paths are not passed users usually expect the action to happen in the current working directory so I suggest using that one instead. You can obtain it by doing `process.cwd()`. Moreover, I recommend normalizing `flags.path` to turn it into an absolute path. You can do it through the [`path.resolve`](https://nodejs.org/api/path.html#pathresolvepaths) method but from the [pathe](https://www.npmjs.com/package/pathe) package instead which is more compatible across OSs.
Like I shared in our sync the other day, I'd abstract away `pathe` by exporting an interface from `@gestaltjs/support`:

```ts
// packages/support/src/index.ts

export { resolve as resolvePath } from "pathe";
```

```ts
import { resolvePath } from "@gestaltjs/support"
```